docs/fiddles/ipc/pattern-1/main.js

```diff
@@ -1,28 +1,24 @@
 const {app, BrowserWindow, ipcMain} = require('electron')
 const path = require('path')
 
+function handleSetTitle (event, title) {
```

Comment:
Why was this moved? This is the same as the previous code, just extracted to a helper function

Reply:
This change was also made to match the [explanations](https://www.electronjs.org/docs/latest/tutorial/ipc#1-listen-for-events-with-ipcmainon) which makes it easier to understand for electron newbies (such as myself).